docs/third-party-developers/extensibility/editor/curating-editor-experience.md

```diff
@@ -0,0 +1,11 @@
+# Curating the Editor Experience
+
+## Disable pattern directory**
+
+To remove patterns provided by WooCommerce Blocks from being accessed in the Inserter, call `remove_theme_support( 'woocommerce-blocks-patterns' );` as a callback to `after_setup_theme` action in your functions.php file. Follow the example:
```

Comment:
Related to the conversation with @nefeline elsewhere, I think it'd be helpful to steer potential assumptions away from this being a theme-only API.
```suggestion
To remove patterns provided by WooCommerce Blocks from being accessed in the Inserter, call `remove_theme_support( 'woocommerce-blocks-patterns' );` in a callback on the `after_setup_theme` action. For example:
```